.travis.yml

```diff
@@ -5,16 +5,13 @@ python:
   - "2.7"
   - "3.4"
   - "3.6"
+  - "3.7"
 before_install:
   - sudo apt-get install liblapack-dev
   - pip install --upgrade pip pytest
-  - pip install wheel cython numpy scipy codecov pytest-cov
-  - if $TRAVIS_PYTHON_VERSION == "3.6"; then
-        pip install scikit-learn;
-    else
-        pip install scikit-learn==0.20.3;
-    fi
+  - pip install wheel cython numpy scipy codecov pytest-cov scikit-learn
```

Comment:
I think now for python versions that are not supported by scikit-learn 0.21 and up, pip will now automatically install the V0.20.3. Let's see if it works